csaw-ctf-2015/forensics/sharpturn-400/README.md

```diff
@@ -14,7 +14,70 @@
 
 ## Write-up
 
-(TODO)
+The git repo was crushed because of a material glitch. So the file sharp.cpp had some wrong inputs in it. To find the flag, start with:
+
+```bash
+$ tar xvf sharpturn.tar.xz
+$ cd shrapturn
+$ git fsck
+Checking object directories: 100% (256/256), done.
+error: sha1 mismatch 354ebf392533dce06174f9c8c093036c138935f3
+error: 354ebf392533dce06174f9c8c093036c138935f3: object corrupt or missing
+error: sha1 mismatch d961f81a588fcfd5e57bbea7e17ddae8a5e61333
+error: d961f81a588fcfd5e57bbea7e17ddae8a5e61333: object corrupt or missing
+error: sha1 mismatch f8d0839dd728cb9a723e32058dcc386070d5e3b5
+error: f8d0839dd728cb9a723e32058dcc386070d5e3b5: object corrupt or missing
+missing blob 354ebf392533dce06174f9c8c093036c138935f3
+missing blob f8d0839dd728cb9a723e32058dcc386070d5e3b5
+missing blob d961f81a588fcfd5e57bbea7e17ddae8a5e61333
+```
+
+Next, create a python script to make fit the SHA keys (See the script.py file in the repo). Use the three blobs in order one at the time  in the SHA variable of the script and run it.
```

Comment:
you didn't add the script.py in your pull request, can you give me the link for it I will add it here